spotify_project/tracks/migrations/0008_rename__duration_ms_track_duration_ms.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 4.0.3 on 2022-04-06 22:30
+
```

Comment:
You probably don't need either of these migrations. If you already ran them, you can reverse them. Which you'll need to do prior to deleting them. Otherwise your DB will be in a wonky state